tensorflow/docs_src/programmers_guide/faq.md

```diff
@@ -285,14 +285,14 @@ more information about the steps for doing this.
 #### What is TensorFlow's coding style convention?
 
 The TensorFlow Python API adheres to the
-[PEP8](https://www.python.org/dev/peps/pep-0008/) conventions.<sup>*</sup> In
-particular, we use `CamelCase` names for classes, and `snake_case` names for
+[PEP8](https://www.python.org/dev/peps/pep-0008/) conventions.*In
```

Comment:
Please revert this. The <sup>*</sup> is an asterisk denoting a footnote, which is referenced at the bottom on line 296, not trying to emphasize the "In particular" phrase.